source/chapters/microphones.rst

```diff
@@ -0,0 +1,315 @@
+.. include:: /shortcuts.rstext
+
+.. _microphones:
+
+Using Microphones
+*****************
+There are three ways to use microphones with Mixxx, each with their own pros
+and cons:
+  * :ref:`Software monitoring<microphones-software-monitoring>`: Cheapest and 
+    simplest to set up, but there is a disorienting delay, referred to as 
+    "latency", between sound going into the microphone and hearing it come out 
+    speakers or headphones.
+  * :ref:`Direct monitoring<microphones-direct-monitoring>`: Recommended for 
+    most users. This is the easiest and cheapest to set up with good sound 
+    quality and without the latency issue of software monitoring.
+  * :ref:`Hardware mixers<microphones-hardware-mixers>`: Most expensive option 
+    to set up with high sound quality. This does not have the latency issue of 
```

Comment:
We do not know the sound quality of an external mixer. 
Can it be better than Mixxx?

Reply:
I don't understand your question. Did you read the whole chapter?

Reply:
The sound quality is not related to microphones-hardware mixers an should be removed from this sentence.